### PR TITLE
TRACE-36: Update docs

### PR DESCRIPTION
**Pull request by BMartinos**
Webpack requires certain modules for loading images into CSS.
The plugin added to docusuarus handles the required modules

### CODE REVIEW

File: docs/api/visualizer/create.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+import TabItem from '@theme/TabItem';
+
+## Create visualizer
+
+To create a new visualizer record you will need to make a TLS request to the OpenHIM API for the below method and endpoint and supply the JSON object for the visualizer record.
+
+
+```curl
+Method: POST
+Endpoint: /visualizers
```
Comment (MattyJ007) [inline docs/api/visualizer/create.md]: Perhaps to be a bit more verbose for new users
```suggestion
Endpoint: {openhim_url}:8080/visualizers
```

File: docs/api/visualizer/create.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    
+      console.log({
+        statusCode: response.statusCode,
+        body
+      })
+    })
+  })()
+  ```
+
+  Execute the below command in your terminal to run the nodejs script
```
Comment (MattyJ007) [inline docs/api/visualizer/create.md]: @BMartinos Perhaps we could include a note to say what directory the user should be in to run this?

File: docs/api/visualizer/delete.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To delete a visualizer record you will need to make a TLS request to the OpenHIM API for the below method and endpoint.
+
+## Delete a specific visualizer record
+
+```curl
+Method: DELETE
+Endpoint: /visualizers/:visualizerId
```
Comment (MattyJ007) [inline docs/api/visualizer/delete.md]: @BMartinos same as above

File: docs/api/visualizer/delete.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  })
+})()
+```
+
+</TabItem>
+<TabItem value="bash">
+
+Ensure that you have created your bash script to construct the HTTP authentication headers and send the request to the OpenHIM API as described in the [authentication section](../introduction/authentication.md). 
+
+Execute the below command in your terminal with the required arguments. Replace the placeholder arguments with the correct implementation details.
```
Comment (MattyJ007) [inline docs/api/visualizer/delete.md]: @BMartinos same as above. perhaps adding the working directory

File: docs/api/visualizer/create.md (hunk 1 above)
Comment (BMartinos) [reply docs/api/visualizer/create.md]: Great thanks @MattyJ007 good suggestion

File: docs/api/visualizer/delete.md (hunk 4 above)
Comment (BMartinos) [reply docs/api/visualizer/delete.md]: done in 5a34e40

File: docs/api/visualizer/delete.md (hunk 3 above)
Comment (BMartinos) [reply docs/api/visualizer/delete.md]: done in 5a34e40

File: docs/api/visualizer/create.md (hunk 2 above)
Comment (BMartinos) [reply docs/api/visualizer/create.md]: done in 5a34e40

File: docs/api/visualizer/create.md (hunk 1 above)
Comment (BMartinos) [reply docs/api/visualizer/create.md]: done in 5a34e40

File: versioned_docs/version-5.2.x/installation/npm.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+If you don’t have ubuntu or prefer to proceed with the installation manually, please follow the following steps.
+
+### Installing the OpenHIM Core
+
+The latest active LTS is **recommended**.
+
+> **Note**: libappstream3 may cause problems with the npm package manager if your ubuntu instance is not fully updated.
+
+1. Install the latest stable Node.js v8 or greater `curl -sL https://deb.nodesource.com/setup_6.x| sudo -E bash` then `sudo apt-get install -y nodejs`
+1. Install and start MongoDB 3.6 or greater. (If you are running Ubuntu 16.04, you may want to configure MongoDB as a systemd service that will automatically start on boot)
```
Comment (MattyJ007) [inline versioned_docs/version-5.2.x/installation/npm.md]: @BMartinos Are there any changes for 18.04? I'm not familiar with systemd

Comment (BMartinos) [reply versioned_docs/version-5.2.x/installation/npm.md]: @MattyJ007 there any any specific changes to systemd in ubuntu 18 as far as im aware. should still work the same

**Pull request merged**